odoo/sql_db.py

```diff
@@ -44,8 +44,8 @@ def undecimalize(value, cr):
 
 real_time = time.time.__call__  # ensure we have a non patched time for query times when using freezegun
 
-re_from = re.compile('.* from "?([a-zA-Z_0-9]+)"? .*$')
-re_into = re.compile('.* into "?([a-zA-Z_0-9]+)"? .*$')
+re_from = re.compile('.* from "?([a-zA-Z_0-9]+)"? .*$', re.MULTILINE)
+re_into = re.compile('.* into "?([a-zA-Z_0-9]+)"? .*$', re.MULTILINE)
```

Comment:
What about JOINs?
And adding `re.IGNORECASE` would make the call to `lower()` useless.